saleor/dashboard/category/urls.py

```diff
@@ -0,0 +1,19 @@
+from django.conf.urls import patterns, url
+
+from . import views
+
+
+urlpatterns = patterns(
```

Comment:
`urlpatterns` can be a regular list now. There is no need to call `patterns(…)`.